pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```diff
@@ -27,7 +24,14 @@ in mkYarnPackage rec {
   packageJSON = ./riot-desktop-package.json;
   yarnNix = ./riot-desktop-yarndeps.nix;
 
-  nativeBuildInputs = [ makeWrapper gtk3 ];
+  nativeBuildInputs = [ wrapGAppsHook ];
+
+  extraBuildInputs = [
```

Comment:
Oh, right: gtk3 is not a native dependency. I don't know what `extraBuildInputs` is exactly, though.

Reply:
mkYarnPackage doesn't have buildInputs exposed apparently.